packages/serve/src/lib/server.ts

```diff
@@ -42,15 +52,42 @@ export class VulcanServer {
     // Create application
     const app = this.container.get<VulcanApplication>(TYPES.VulcanApplication);
     await app.useMiddleware();
-    await app.buildRoutes(schemas, this.config.types);
+    await app.buildRoutes(schemas, this.config['types']);
     // Run server
-    const server = http.createServer(app.getHandler()).listen(port);
-    this.server = server;
-    return server;
+    this.servers = this.runServer(app, port, httpsPort);
+    return this.servers;
   }
-
   public async close() {
-    if (this.server) this.server.close();
+    if (this.servers) {
+      if (this.servers['http']) this.servers['http'].close();
+      if (this.servers['https']) this.servers['https'].close();
+      this.servers = undefined;
+    }
     this.container.unload();
   }
+
+  /**
+   * Run server for https when config has setup ssl and middleware 'enforce-https' enabled with LOCAL type, or keep http
+   */
+  private runServer(app: VulcanApplication, port: number, httpsPort: number) {
+    const options = getEnforceHttpsOptions(this.config['enforce-https']);
+    if (options && this.config.ssl) {
+      const options = {
+        key: fs.readFileSync(this.config.ssl.keyFile),
+        cert: fs.readFileSync(this.config.ssl.certFile),
+      };
```

Comment:
We should provide "ca" property for CA bundle/chain.
Ref
https://medium.com/@superseb/get-your-certificate-chain-right-4b117a9c0fce
https://www.namecheap.com/support/knowledgebase/article.aspx/9705/33/installing-an-ssl-certificate-on-nodejs/

Reply:
Thanks @oscar60310 for reviewing and suggestion, also thanks for sharing the document to let me know we should also provide CA bundle chain!